com.unity.render-pipelines.high-definition/Editor/AssetProcessors/FBXArnoldSurfaceMaterialDescriptionPreprocessor.cs

```diff
@@ -1,7 +1,10 @@
 using System.IO;
 using UnityEditor.AssetImporters;
 using UnityEngine;
+#if !UNITY_2020_2_OR_NEWER
```

Comment:
should be

#if UNITY_2020_2_OR_NEWER
using UnityEditor.AssetImporters;
#else
using UnityEditor.Experimental.AssetImporters;	using UnityEditor.Experimental.AssetImporters;
#endif

Reply:
same for other stuff

Reply:
There's already a 'using UnityEditor.AssetImporters;' above in this file, should be alright, no ? I'm getting my hands on a Unity build of the branch that does the migration to confirm..

Reply:
that's exactly my point, we should move the one above in the gaurd band, I will fix it :)

Reply:
done